src/js/providers/html5.ts

```diff
@@ -179,6 +179,9 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         },
 
         canplay(this: ProviderWithMixins): void {
+            if (_this.seeking) {
+                return;
+            }
```

Comment:
Thinking about it more, I think we should make this check `_canSeek`, because that will ensure this cannot continue multiple times. `_this.seeking` will only exit with a player initiated seek, and won't prevent this bug in scenarios not initiated by a seek (like if it happens on play with some odd HLS streams)

Reply:
Good call, switched it over